docker/backend-dev/conf/Config2.pm

```diff
@@ -66,13 +76,20 @@ $memd_servers = [ "memcached:11211" ];
 $facebook_app_id = "";
 $facebook_app_secret = "";
 
+$google_cloud_vision_api_key = $ENV{GOOGLE_CLOUD_VISION_API_KEY};
+$crowdin_project_identifier = $ENV{CROWDIN_PROJECT_IDENTIFIER};
+$crowdin_project_key = $ENV{CROWDIN_PROJECT_KEY};
+$csrf_secret = $ENV{CSRF_SECRET};
```

Comment:
We actually removed CSRF in #1362, so maybe this config setting, and the corresponding environment variable should be deleted.